public/src/main/java/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -75,3 +77,30 @@ interface StoreProduct {
     )
     val sku: String
 }
+
+interface PlatformProductId {
```

Comment:
Would be good moving all these to a different file. And since it's public, add documentation

Reply:
Actually, does this need to be public? Seems like this could be an internal extension of GoogleStoreProduct and AmazonStoreProduct